installlib/installcommand.class.php

```diff
@@ -455,6 +463,17 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$sbin = \FreePBX::Config()->get("AMPSBIN");
 		$bin = \FreePBX::Config()->get("AMPBIN");
 
+		$output->writeln("bin is: $bin");
```

Comment:
nice catch! I've seen people having these errors and didn't understand why. doh'